src/components/elements/CommonToggle.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import { useArgs } from '@storybook/preview-api';
+import { Meta, StoryObj } from '@storybook/react';
+import CommonToggle from './CommonToggle';
+
+export default {
+  component: CommonToggle,
+} as Meta<typeof CommonToggle>;
+
+type Story = StoryObj<typeof CommonToggle>;
+
+export const Default: Story = {
+  args: {
+    items: [
+      { value: 'YES', label: 'Yes' },
+      { value: 'NO', label: 'No' },
+    ],
+    value: 'YES',
+  },
+  // https://github.com/storybookjs/storybook/discussions/21680#discussioncomment-7698496
+  render: function Component(args) {
```

Comment:
Other similar implementations:
https://github.com/greenriver/hmis-frontend/blob/eb66d3a63cfb82a8e316fa0c4580957ff94b5db9/src/components/elements/input/MultiFieldInput.stories.tsx#L10-L15
and
https://github.com/greenriver/hmis-frontend/blob/eb66d3a63cfb82a8e316fa0c4580957ff94b5db9/src/components/elements/input/SsnInput.stories.tsx#L13-L18

On those we define the render function on the default export, rather than on wone of the `Story`s. That lets you make multiple stories without rewriting the render function (as in `MultiFieldInput` which has several). Maybe we should stick to that pattern? And I think that line disabling eslint is unnecessary, lint works fine without it.